jirate/jira_cli.py

```diff
@@ -476,19 +505,40 @@ def create_from_template(args):
         raise e
 
     for issue in template['issues']:
+        reserved_fields = ['subtasks']
+        required_fields = ['summary']
+        translate_fields = {'issue_type': 'issuetype'}
+
+        # These may be overridden
+        start_fields = {'issuetype': 'Task', 'project': args.project.project_name}
+
+        creation_fields = _parse_creation_args(issue, required_fields, reserved_fields, translate_fields, start_fields)
+
+        # Cache all metadata now (so we can debug subtask creation if needed.
+        issuetype = creation_fields['issuetype']
+        if issuetype not in metadata_by_type:
+            metadata_by_type[issuetype] = _metadata_by_type(args.project, issuetype)
+        if 'subtasks' in issue and issue['subtasks']:
+            metadata_by_type[_subtask] = _metadata_by_type(args.project, _subtask)
+        metadata = metadata_by_type[issuetype]
+
         filed = {}
-        if 'description' not in issue:
-            issue['description'] = ""
-        parent = args.project.new(issue['summary'], description=issue['description'], issue_type=issue['issue_type'])
-        filed['parent'] = parent.raw['key']
+        # args.project.jira = fake_jira()  # debugging
+        parent = args.project.create(metadata['fields'], **creation_fields)
+        filed['parent'] = parent.key
 
-        if issue['subtasks']:
+        if 'subtasks' in issue and issue['subtasks']:
+            # Set once
+            metadata = metadata_by_type[_subtask]
             filed['subtasks'] = []
             for subtask in issue['subtasks']:
-                if 'description' not in subtask:
-                    subtask['description'] = ""
-                child = args.project.subtask(parent.raw['key'], subtask['summary'], subtask['description'])
-                filed['subtasks'].append(child.raw['key'])
+                reserved_fields = ['subtasks', 'issuetype', 'parent']
+                # required_fields and translate_fields is same
+                start_fields = {'issuetype': _subtask, 'parent': parent.key}
```

Comment:
Need to verify that start_fields['parent'] is not pruned by reserved_fields['parent']

Reply:
Test added for this.